<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Caesar</class>
 <widget class="QDialog" name="Caesar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>498</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Caesat cipher</string>
  </property>
  <widget class="QTableView" name="tableView1">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>-10</y>
     <width>481</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView2">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>481</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>181</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>14</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Encryption</string>
   </property>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>10</y>
     <width>161</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>14</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Decryption</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>60</y>
     <width>81</width>
     <height>431</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>false</bold>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>421</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>12</pointsize>
     <italic>false</italic>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Text to encrypt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_encrypt">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>240</y>
     <width>110</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ENCRYPT</string>
   </property>
  </widget>
  <widget class="QLabel" name="label5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Resoult</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>421</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>12</pointsize>
     <italic>false</italic>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label8">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>320</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Resoult</string>
   </property>
  </widget>
  <widget class="QLabel" name="label6">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>80</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Text to Decrypt</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>100</y>
     <width>421</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>12</pointsize>
     <italic>false</italic>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_decrypt">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>240</y>
     <width>110</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>DECRYPT</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit4">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>340</y>
     <width>421</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>12</pointsize>
     <italic>false</italic>
    </font>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>91</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label4">
      <property name="font">
       <font>
        <family>Bahnschrift</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Shift</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit1">
      <property name="font">
       <font>
        <family>Bahnschrift</family>
        <pointsize>12</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>240</y>
     <width>91</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label7">
      <property name="font">
       <font>
        <family>Bahnschrift</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Shift</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit2">
      <property name="font">
       <font>
        <family>Bahnschrift</family>
        <pointsize>12</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
